config/v1/0000_10_config-operator_01_infrastructure.crd.yaml

```diff
@@ -56,6 +56,29 @@ spec:
                       description: AWS contains settings specific to the Amazon Web Services infrastructure provider.
                       type: object
                       properties:
+                        resourceTags:
+                          description: ResourceTags is a list of additional tags to apply to AWS resources created for the cluster. See https://docs.aws.amazon.com/general/latest/gr/aws_tagging.html for information on tagging AWS resources. AWS supports a maximum of 50 tags per resource. OpenShift reserves 25 tags for its use, leaving 25 tags available for the user. While ResourceTags field is mutable, items can not be removed.
+                          type: array
+                          maxItems: 25
```

Comment:
The [Update enhancement proposal to support update of user-defined AWS tags](https://github.com/openshift/enhancements/pull/1051) states that:
> OpenShift is bound by the AWS resource that allows the fewest number of tags. An AWS S3 bucket can have at most 10 tags.

Reply:
Also, it states that OpenShift reserves 5 of those for its own use, so the maxItems here should be 5